cmake/modules/AddSwift.cmake

```diff
@@ -294,8 +294,12 @@ function(_add_variant_swift_compile_flags
     sdk arch build_type enable_assertions result_var_name)
   set(result ${${result_var_name}})
 
+  # On Windows, we don't set SWIFT_SDK_WINDOWS_PATH, so don't include it
```

Comment:
The compiler behaves differently when there's an SDK provided and when there isn't. Is there nothing reasonable to default this to?